test/DefaultPrompt.Tests.ps1

```diff
@@ -53,8 +49,8 @@ Describe 'Default Prompt Tests' {
         }
         It 'Returns the expected prompt string with prefix, suffix and abbrev home set' {
             Set-Location $Home -ErrorAction Stop
-            $GitPromptSettings.DefaultPromptPrefix = '[$(hostname)] '
-            $GitPromptSettings.DefaultPromptSuffix = ' - $(6*7)> '
+            $GitPromptSettings.DefaultPromptPrefix.Text = '[$(hostname)] '
+            $GitPromptSettings.DefaultPromptSuffix.Text = ' - $(6*7)> '
```

Comment:
If we're assuming no colors, is there a downside to setting the property directly (meaning a new span is created from the string constructor) instead of setting `Text`?

Reply:
No, just a minor perf hit no one would likely notice.